api/client/client.go

```diff
@@ -1962,20 +1962,31 @@ func (c *Client) DeleteTrustedCluster(ctx context.Context, name string) error {
 	return trail.FromGRPC(err)
 }
 
+func (c *Client) getTokenV2(ctx context.Context, name string) (types.ProvisionToken, error) {
+	resp, err := c.grpc.GetToken(ctx, &types.ResourceRequest{Name: name}, c.callOpts...)
+	if err != nil {
+		return nil, trail.FromGRPC(err)
+	}
+	return resp, nil
+}
+
 // GetToken returns a provision token by name.
 func (c *Client) GetToken(ctx context.Context, name string) (types.ProvisionToken, error) {
```

Comment:
should we emphasize that `GetToken` can return two types of tokens V3 and V2? I think it might be useful for typecasts